test/specs/smooch.spec.js

```diff
@@ -282,16 +282,14 @@ describe('Smooch', () => {
                             }));
                         });
 
-                        it('should throw', () => {
-                            (function() {
-                                Smooch.init(props);
-                            }).should.throw(/already initialized/);
+                        it('should reject', () => {
+                            Smooch.init(props).should.be.rejectedWith(Error, /already initialized/);
```

Comment:
I think you'll need to `return` here, `rejectWith` should return a promise which mocha should wait on. Same with the tests below, `return` or `yield`